.asf.yaml

```diff
@@ -59,6 +59,7 @@ github:
     - abh1sar
     - FelipeM525
     - lucas-a-martins
+    - nicoschmdt
```

Comment:
we can delete @rajujith and @winterhazel , @nicoschmdt . Could you include that in youe PR please?